3_visualize/src/plot.r

```diff
@@ -0,0 +1,48 @@
+library(dplyr)
```

Comment:
Nice work paring down the library lists in each file - that helps with identifying package dependencies for the project overall. In this file and in 2_process/src/process.r, I do see calls to `readr::read_csv`, so `library(readr)` should be included in this top block to declare that additional dependency. (I'm also not above calling `library(tidyverse)` these days instead of picking out individual tidyverse dependencies to declare, since they're all so often useful. Personal preference there.)

Reply:
Thanks, and done.